packs/feats/tide-hardened.json

```diff
@@ -39,7 +39,7 @@
                     }
                 ],
                 "selector": "saving-throw",
-                "text": "When you roll a success on a save against a cold or water effect, you get a critical success instead.",
+                "text": "PF2E.SpecificRule.Undine.TideHardened.Note",
```

Comment:
We've been moving away from Note REs in favor of AdjustDegreeOfSuccess. Consider removing the Note entirely.